src/language-server/auto-completer.ts

```diff
@@ -114,6 +115,7 @@ export default class AutoCompleter extends Handler {
       if (location.kind === 'attribute') {
         return this.getAttributeCompletions(document, location);
       }
+      return this.getStandardSnippetCompletions();
```

Comment:
Hm, should explicitly check for the location being inside of a script section, as we might add other location kinds.

For bonus points it would be cool to do to JS what we did for CSS, and have an entire class of result for locResult.language === `js` but that might be more than you're interested in doing right now.

Reply:
I left a TODO, as I tried to make that work, but it required new AST traversal methods in `ast-from-source-position`, which seemed too much for this PR. Should be fixed someday though :smile: